src/session_impl.cpp

```diff
@@ -5700,7 +5700,7 @@ namespace {
 
 #ifndef TORRENT_DISABLE_LOGGING
 		session_log("starting DHT, running: %s, router lookups: %d, aborting: %s"
-			, m_dht ? "true" : "false", m_outstanding_router_lookups
+			, m_dht ? "true" : "false", m_outstanding_router_lookups);
```

Comment:
you added an extra ");" here.
syntax error: https://ci.appveyor.com/project/arvidn/libtorrent/build/8105/job/s6l498mo8axjsclp#L243